test/test_vmap.py

```diff
@@ -3746,49 +3754,206 @@ def test_like_functions(self, device, randomness, batched_input):
 
     @parametrize('use_generator', [True, False])
     @parametrize('randomness', ['error', 'same', 'different'])
-    def test_random_inplace_not_batched(self, device, use_generator, randomness):
-        # tests that when in place random ops are being called during vmap but not with a batched tensor,
-        # it behaves like a normal random_ when using same random and errors during different random
-        supported_ops = [
+    @parametrize('batched_input', [True, False])
+    def test_random_inplace_single(self, device, use_generator, randomness, batched_input):
```

Comment:
nit: test_random_unary_inplace: "unary" is shorthand for "single tensor input"

Reply:
Fixed